snapcraft/tests/fixture_setup.py

```diff
@@ -405,6 +406,89 @@ def call_effect(*args, **kwargs):
     return call_effect
 
 
+class FakeAptGetBuildDep(fixtures.Fixture):
+    '''Mock apt-get build-dep output'''
+
+    def __init__(self, packages, arch='', update_error=False,
+                 not_cached=False, not_available=False):
+        self.filename = '{}/{}abcdef.dsc'.format(tempfile.gettempdir(),
+                                                 tempfile.gettempprefix())
+        if arch:
+            arch = ':{}'.format(arch)
+        prolog = '''NOTE: This is only a simulation!
```

Comment:
mind using `dedent` here? `from textwrap import dedent` (look at the new-cli branch occurrences for quick inspiration)